tests/integration/conftest.py

```diff
@@ -1,565 +1,563 @@
+#                         "label": "Primary IP Address",
+#                         "optional": True,
+#                         "cardinality": "one",
+#                         "kind": "Attribute",
+#                     }
+#                 ],
+#             },
+#         ],
+#     }
+
+#     return SCHEMA
+
+
+# @pytest.fixture(scope="module")
+# async def hierarchical_schema() -> dict:
+#     schema = {
+#         "version": "1.0",
+#         "generics": [
+#             {
+#                 "name": "Generic",
+#                 "namespace": "Location",
+#                 "description": "Generic hierarchical location",
+#                 "label": "Location",
+#                 "hierarchical": True,
+#                 "human_friendly_id": ["name__value"],
+#                 "include_in_menu": True,
+#                 "attributes": [
+#                     {"name": "name", "kind": "Text", "unique": True, "order_weight": 900},
+#                 ],
+#             }
+#         ],
+#         "nodes": [
+#             {
+#                 "name": "Country",
+#                 "namespace": "Location",
+#                 "description": "A country within a continent.",
+#                 "inherit_from": ["LocationGeneric"],
+#                 "generate_profile": False,
+#                 "default_filter": "name__value",
+#                 "order_by": ["name__value"],
+#                 "display_labels": ["name__value"],
+#                 "children": "LocationSite",
+#                 "attributes": [{"name": "shortname", "kind": "Text"}],
+#             },
+#             {
+#                 "name": "Site",
+#                 "namespace": "Location",
+#                 "description": "A site within a country.",
+#                 "inherit_from": ["LocationGeneric"],
+#                 "default_filter": "name__value",
+#                 "order_by": ["name__value"],
+#                 "display_labels": ["name__value"],
+#                 "children": "",
+#                 "parent": "LocationCountry",
+#                 "attributes": [{"name": "shortname", "kind": "Text"}],
+#             },
+#         ],
+#     }
+#     return schema
+
+
+# class BusRecorder(InfrahubMessageBus):
+#     def __init__(self, component_type: Optional[ComponentType] = None):
+#         self.messages: list[InfrahubMessage] = []
+#         self.messages_per_routing_key: dict[str, list[InfrahubMessage]] = {}
+
+#     async def publish(
+#         self, message: InfrahubMessage, routing_key: str, delay: Optional[MessageTTL] = None, is_retry: bool = False
+#     ) -> None:
+#         self.messages.append(message)
+#         if routing_key not in self.messages_per_routing_key:
+#             self.messages_per_routing_key[routing_key] = []
+#         self.messages_per_routing_key[routing_key].append(message)
+
+#     @property
+#     def seen_routing_keys(self) -> list[str]:
+#         return list(self.messages_per_routing_key.keys())
```

Comment:
are you hoping to uncomment this file in a later PR or should it be deleted?

Reply:
we need to finish the conversion of all the integration tests, probably best to keep it for now